docs/pages/Cardinal/API-Reference/World.mdx

```diff
@@ -0,0 +1,529 @@
+```
+#### Example
+
+```go
+package main
+
+import (
+	"log"
+
+    "pkg.world.dev/world-engine/cardinal/ecs"
+
+    "github.com/my-username/my-world-engine-project/components"
+)
+
+func main() {
+    // ... world setup ...
+
+    err := world.RegisterComponents(
+		components.LocationComponent,
+		components.AttackPowerComponent,
+		components.HealthComponent,
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+```
+
+#### Parameters
+
+| Parameter    | Type                    | Description                                                    |
+|--------------|-------------------------|----------------------------------------------------------------|
+| components   | ...component.IComponentType | Variadic parameter for components to be registered with the World. |
+
+#### Return Value
+
+| Type | Description                                                                  |
+|-------------|------------------------------------------------------------------------------|
+| error       | An error indicating any issues that occurred during the component registration. |
+
+
+### RegisterReads
+
+`RegisterReads` registers the reads in the `World`. This allows the `Read` endpoints to be automatically generated.
+
+```go
+func (w *World) RegisterReads(reads ...IRead) error
+```
+#### Example
+
+```go
+package main
+
+import (
+	"log"
+
+    "pkg.world.dev/world-engine/cardinal/ecs"
+
+    "github.com/my-username/my-world-engine-project/reads"
+)
+
+func main() {
+    // ... world setup ...
+
+    err := world.RegisterReads(
+		reads.ReadLocation,
+		reads.ReadHealth,
+		reads.ReadAttackPower,
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+```
+#### Parameters
+| Parameter    | Type       | Description                                                   |
+|--------------|------------|---------------------------------------------------------------|
+| reads        | ...IRead   | Variadic parameter for IRead instances to be registered.      |
+
+
+#### Return Value
+| Type   | Description                                                               |
+|--------|---------------------------------------------------------------------------|
+| error  | An error indicating any issues that occurred during the read registration. |
+
+### RegisterTransactions
+
+`RegisterTransactions` registers the transactions in the `World`. This allows transaction endpoints to be automatically generated.
+
+<Callout type={"warning"}>
+    `RegisterTransactions` can be only be called once. Subsequent calls to the method will return an error.
+</Callout>
+
+```go
+func (w *World) RegisterTransactions(txs ...transaction.ITransaction) error
+```
+
+#### Example
+```go
+package main
+
+import (
+	"log"
+
+    "pkg.world.dev/world-engine/cardinal/ecs"
+
+    "github.com/my-username/my-world-engine-project/txs"
+)
+
+func main() {
+    // ... world setup ...
+
+    err := world.RegisterTransactions(
+		txs.MoveTx,
```

Comment:
package name in starter-game-template is singular `tx` would like to keep it consistent

Reply:
[package consistency:](https://github.com/Argus-Labs/world-engine/pull/200/commits/3ace678c1ac3dfe79a376c766e89708b80e22404)